lib/module-index-server/src/routes/upsert_module_route.rs

```diff
@@ -121,11 +121,13 @@ pub async fn upsert_module_route(
         SiPkgKind::Module => ModuleKind::Module,
     };
 
-    let new_schema_id = Some(SchemaId(Ulid::new()));
+    let new_schema_id = Some(SchemaId::new());
     let schema_id = match module_kind {
         ModuleKind::WorkspaceBackup => None,
         ModuleKind::Module => match module_schema_id {
-            Some(schema_id_string) => Some(SchemaId(Ulid::from_string(&schema_id_string)?)),
+            Some(schema_id_string) => {
+                Some(SchemaId::from_raw_id(Ulid::from_string(&schema_id_string)?))
```

Comment:
Why a `from_raw_id` `fn` instead of a `impl From<ulid::Ulid> for SchemaId`? (Pretty much the same question for the other `from_raw_id`, too.)

Reply:
Just an oversight! Fixing...